tools/osbuilder/rootfs-builder/cbl-mariner/config.sh

```diff
@@ -5,7 +5,8 @@
 OS_NAME=cbl-mariner
 OS_VERSION=${OS_VERSION:-2.0}
 LIBC="gnu"
-PACKAGES="core-packages-base-image ca-certificates device-mapper"
+PACKAGES="kata-packages-uvm"
+[ "$CONF_GUEST" = yes ] && PACKAGES+=" kata-packages-coco-uvm"
```

Comment:
Do you know where CONF_GUEST=yes is defined?

Reply:
It would be introduced as a new variable in[CBL-Mariner/SPECS/kata-containers-cc/mariner-coco-build-uvm.sh ](https://github.com/microsoft/CBL-Mariner/blob/mahuber/sprt/katacc-new-branch/SPECS/kata-containers-cc/mariner-coco-build-uvm.sh)(line 18)